README.md

```diff
@@ -17,11 +20,12 @@ import "gopkg.in/d4l3k/messagediff.v1"
 type someStruct struct {
   A, b int
   C []int
+  D int `testdiff:"ignore"`
 }
 
 func main() {
-			a := someStruct{1, 2, []int{1}}
-			b := someStruct{1, 3, []int{1, 2}}
+			a := someStruct{1, 2, []int{1}, 9}
+			b := someStruct{1, 3, []int{1, 2}, 10}
```

Comment:
Probably should add a separate example to show off this feature since most people won't care about it.